Dockerfile.testing

```diff
@@ -0,0 +1,18 @@
+# Dockerfile for usaspending-test image
+
+# If any changes are made to the usaspending-backend image (Dockerfile) then this image needs to be rebuilt to stay in sync.
+
+# If you do not already have the `usaspending-backend` image locally you can build it by running:
+#       docker compose --profile usaspending build
+# in the parent usaspending-api directory.
+
+FROM usaspending-backend:latest
+
+WORKDIR /usaspending-api
+
+# Prevent a Pytest RuntimeError where it expects the parent directory to be the name of an installed Django app
+# This will cause a RuntimeError because `dockermount` isn't the name of an installed Django app, so we move the files
+#   to a `/usaspending-api` parent directory which is the name of an installed Django app.
+RUN mv /dockermount/* /usaspending-api/
```

Comment:
I don't feel like this should be necessary. `usaspending-api` I don't believe is either our django "project" or one of it's "apps". Maybe `usaspending_api`?  What's the error you get? It may just be a matter of:
1) setting the environment variable `DJANGO_SETTINGS_MODULE`. You can do that in `Dockerfile` with
```ENV DJANGO_SETTINGS_MODULE=usaspending_api.settings```

2) Making sure the `PYTHONPATH` is resolving all dependencies where the pip install is happening. The place where `pytest` is invoked should be added to the `PYTHONPATH` automatically, but make sure that that place is set as your `WORKDIR`.

Also, if you wanted to name it something other than `dockermount` consider what we do in the AMI images for the api backend: `/data-act/backend/usaspending-api`

Honestly, we should just move where the files are mounted in the base `usaspending-backend` image to get rid of the oddly named "dockermount" directory.

Reply:
This is the error message from `pytest`:

```
RuntimeError: Model class dockermount.usaspending_api.etl.tests.integration.test_model.TestModel doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```

Reply:
Setting the `DJANGO_SETTINGS_MODULE=usaspending_api.settings` did not fix the error. I'll double check the `PYTHONPATH`. I'm pretty sure it's set correctly, but I'll verify.

Reply:
I left this line in the Dockerfile.testing file, because I haven't been able to find a way to prevent `pytest` from throwing this error whenever the files are in a parent directory with a name other than **usaspending-api**